kalite/main/content_rating_models.py

```diff
@@ -5,14 +6,18 @@
 from securesync.models import DeferredCountSyncedModel
 
 class ContentRating(DeferredCountSyncedModel):
+
+    minversion = "0.15.0"
+
     class Meta:
         unique_together = ("content_kind", "content_id", "user")
 
     # Maintain info on content type -- could be video, exercise, etc, but we should be able to uniquely id it
     content_kind = models.CharField(max_length=100, db_index=True, blank=False)
     content_id = models.CharField(max_length=100, db_index=True, blank=False)
+    content_source = models.CharField(max_length=100, db_index=True, default=settings.CHANNEL)

```

Comment:
Added the following to the Release Workflow:

> Squashing migrations
> With South: Delete all migrations that were added since the last release. Create one single migration for each app.